packages/core/src/release.ts

```diff
@@ -116,6 +116,24 @@ export const defaultLabels: ILabelDefinition[] = [
     description: "Changes only affect the documentation",
     releaseType: "none",
   },
+  {
+    name: "tests",
+    changelogTitle: "🧪 Tests",
+    description: "Add or improve existing tests",
+    releaseType: "none",
+  },
+  {
+    name: "dependencies",
+    changelogTitle: "🔩 Dependency Updates",
+    description: "Update one or more dependencies version",
+    releaseType: "none",
+  },
+  {
+    name: "performance",
+    changelogTitle: "🏎 Performance",
+    description: "Improve performance of an existing feature",
+    releaseType: "none",
```

Comment:
This should probably be a `patch` right? You would want a release to go out

Reply:
Ok, changed. Do you think `dependencies` should also be a `patch`?

Reply:
I'd say that kind of depends on the user. I like that it has a `none` release type so a can merge a bunch quickly.

Reply:
Ok, cool. Then, as I see it, all changes are made.